Jellyfin.Plugin.Anime/Providers/AniDB/Identity/AniDbTitleDownloader.cs

```diff
@@ -18,7 +18,8 @@ public class AniDbTitleDownloader : IAniDbTitleDownloader
         /// <summary>
         /// The URL for retrieving a list of all anime titles and their AniDB IDs.
         /// </summary>
-        private const string TitlesUrl = "http://anidb.net/api/animetitles.xml.gz";
+        private const string TitlesUrl = "http://anidb.net/api/anime-titles.xml.gz";
+        private const string UserAgent = "mediabrowser";
```

Comment:
```suggestion
        private const string UserAgent = "Jellyfin";
```

No idea if we need to register it with them, but we should announce ourselves as who we are, not an old name for something that doesn't exist.

Reply:
I would make this lowercase though if we can change it.

Reply:
Yeah, I'd prefer to send jellyfin, and in this case I think we can. I chose "mediabrowser" to match the client name [here](https://github.com/jellyfin/jellyfin-plugin-anime/blob/master/Jellyfin.Plugin.Anime/Providers/AniDB/Metadata/AniDbSeriesProvider.cs#L30), which does have to be registered

Reply:
Oh, in that case I would use a single const for both cases.

Reply:
We should probably ask @anthonylavado to go talk to the AniDB guys and register us, so we appear in the apps list on AniDB and have our own thing.

Reply:
Looks like that's exactly what #46 is about.

Re: User-Agent, if we decide to change to "jellyfin", perhaps we could be more descriptive. i.e.
```
UserAgent = "jellyfin-plugin-anime/" + Assembly.GetExecutingAssembly().GetName().Version.ToString();
```
to send `jellyfin-plugin-anime/5.0.0`

Reply:
I think `jellyfin-plugin-anime` would be sufficient since we don't push updates often.

Reply:
Having the version could be useful if AniDB ever needs to block one of our versions for one reason or another, though.

But we need to check with them if they need a unique one or if they can accommodate versioned agents.